### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by TarasYurkovskyi**
image

### CODE REVIEW

**Review by BohdanTrue — APPROVED** [review]
GJ

File: src/test/java/com/example/bookstore/service/impl/ShoppingCartServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    private static UpdateCartItemDto updateCartItemDto;
+
+    @Mock
+    private ShoppingCartRepository shoppingCartRepository;
+    @Mock
+    private CartItemRepository cartItemRepository;
+    @Mock
+    private ShoppingCartMapper shoppingCartMapper;
+    @InjectMocks
+    private ShoppingCartServiceImpl shoppingCartService;
```
Comment (yevheniisynytsia) [inline src/test/java/com/example/bookstore/service/impl/ShoppingCartServiceImplTest.java:49]: good order of fields 👍

File: src/test/java/com/example/bookstore/service/impl/ShoppingCartServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+        updateCartItemDto = new UpdateCartItemDto()
+                .setQuantity(5);
+    }
+
+    @Test
+    @DisplayName("Get shopping cart")
+    void getShoppingCart_ValidEmail_ReturnShoppingCartDto() {
+
+        Mockito.when(shoppingCartRepository.findByEmail(user.getEmail()))
```
Comment (yevheniisynytsia) [inline src/test/java/com/example/bookstore/service/impl/ShoppingCartServiceImplTest.java:111]: please add static import

File: src/test/resources/database/add-shopping-carts.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+INSERT INTO shopping_carts (id, user_id) VALUES (1, 1)
```
Comment (yevheniisynytsia) [inline src/test/resources/database/add-shopping-carts.sql:1]: add empty lines

**Review by yevheniisynytsia — APPROVED** [review]
Great job! Left minor comments, so approving

**Pull request merged**